wikipendium/wiki/templates/article.html

```diff
@@ -15,10 +15,13 @@
 {% endblock %}
 
 {% block content %}
+{% autoescape off %}
+    {{ toc }}
+{% endautoescape %}
+<a href="javascript:;" id="toggle-toc">‹</a>
```

Comment:
I guess I could have omitted the href altogether and applied cursor: pointer with css instead.